.github/workflows/task-test.yml

```
@@ -85,6 +89,46 @@ jobs:
     with:
       platform: ${{ inputs.platform }}
       architecture: ${{ inputs.architecture }}
+
+  # Start EC2 runner for self-hosted platforms
+  start-runner:
+    name: Start self-hosted EC2 runner
+    if: ${{ inputs.self-hosted }}
+    runs-on: ubuntu-latest
+    env:
+      AWS_IMAGE_ID: ami-09fabd03bb09b3704
+      AWS_INSTANCE_TYPE: c7i.xlarge
+      TAGS: | # Make sure there is no trailing comma!
+        [
+        {"Key": "Name",         "Value": "redisearch-ci-runner"},
+        {"Key": "Environment",  "Value": "CI"},
+        {"Key": "Run ID",       "Value": "${{ github.run_id }}"},
+        {"Key": "PR",           "Value": "${{ github.event.number }}"},
+        {"Key": "Owner",        "Value": "${{ github.actor }}"},
+        {"Key": "Project",      "Value": "${{ github.repository }}"}
+        ]
```

For better string safety:
```suggestion
        [
          {"Key": "Name",         "Value": "redisearch-ci-runner"},
          {"Key": "Environment",  "Value": "CI"},
          {"Key": "Run ID",       "Value": ${{ toJson(github.run_id) }}},
          {"Key": "PR",           "Value": "${{ github.event.number }}"},
          {"Key": "Owner",        "Value": ${{ toJson(github.actor) }}},
          {"Key": "Project",      "Value": ${{ toJson(github.repository) }}}
        ]
```